examples/soap-calculator/src/services/calculator.service.ts

```diff
@@ -0,0 +1,31 @@
+import {getService, juggler} from '@loopback/service-proxy';
+import {inject, Provider} from '@loopback/core';
+import {CalculatorDSDataSource} from '../datasources/calculator-ds.datasource';
+
+export interface CalculatorResult {
+  result: {
+    [param: string]: number;
```

Comment:
It would be nicer to have one interface for each operation result, e.g.

```ts
export interface AddResult {
  result: {AddResult: number}
}

export interface MultiplyResult {
  result: {MultiplyResult: number}
}

// usage
export interface CalculatorService {
  Multiply(args: CalculatorParameters): Promise<MultiplyResult>;
  // ...
}
```

Unless I am missing something and there is a good reason why we cannot do what I proposed. What do you think, @marioestradarosa?

Reply:
Agree, since that can show that multiple methods from the same connector can have different response structures and the code will be able to handle them separately.